### PR TITLE
i18n(zh-cn): translate guides/cms/microcms.mdx

### PR DESCRIPTION
**Pull request by Lilian97**
#### What kind of changes does this PR include?

- Translated content

#### Description

- translate guides/cms/microcms.mdx

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/microcms.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: microCMS & Astro
+description: 使用 microCMS 向你的 Astro 项目添加内容
+type: cms
+stub: true
+service: microCMS
+i18nReady: true
+---
+
+[microCMS](https://microcms.io/)是一个基于 API 的无头 CMS，允许你使用模式定义内容，并使用仪表板管理内容。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/microcms.mdx:10]:
```suggestion
[microCMS](https://microcms.io/) 是一个基于 API 的无头（headless） CMS，允许你使用模式定义内容，并使用仪表板管理内容。
```

File: src/content/docs/zh-cn/guides/cms/microcms.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## 官方资源
+
+- 查看[microCMS 官方文档](https://document.microcms.io/tutorial/astro/astro-top)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/microcms.mdx:14]:
```suggestion
- 查看 [microCMS 官方文档](https://document.microcms.io/tutorial/astro/astro-top)
```

File: src/content/docs/zh-cn/guides/cms/microcms.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- 博客：[使用 microCMS 建立博客](https://blog.microcms.io/astro-microcms-introduction/)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/microcms.mdx:15]:
```suggestion
- 博客：[使用 microCMS 搭建博客](https://blog.microcms.io/astro-microcms-introduction/)
```

**Review by liruifengv — APPROVED** [review]
A little change. LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**